config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
Oh yeah, I see what you mean since the method's only defined in the module.  How about http://paste.fedoraproject.org/235780/35071618/?  It defines the method in a module which'll get included after `audited` (though perhaps add a `require` to be certain).